pallets/admin-utils/src/lib.rs

```diff
@@ -26,7 +26,7 @@ pub mod pallet {
 
     /// Configure the pallet by specifying the parameters and types on which it depends.
     #[pallet::config]
-    pub trait Config: frame_system::Config {
+    pub trait Config: frame_system::Config + pallet_subtensor::Config {
```

Comment:
For TotalSubnetLocked, we can set it via a function as other hyperparameter. Why introduce the pallet_subtensor::Config this time. is anything special?

Reply:
If it is a temp solution, we can easily remove it later?

Reply:
It's the idiomatic way to achieve this type of pallet coupling in Substrate, it allows us to directly access items from the subtensor pallet without needing all the boilerplate of specifying and implementing the interface. Notice I didn't need any additional boilerplate in the new fns here, I can just direct access pallet_subtensor storage items.